UI/window-basic-main.hpp

```diff
@@ -1331,6 +1330,106 @@ public slots:
+			     bool useWizard = false);
+	void SetupDuplicateProfile(const std::string &profileName);
+	void SetupRenameProfile(const std::string &profileName);
+
+	const OBSProfile &CreateProfile(const std::string &profileName);
+	void RemoveProfile(OBSProfile profile);
+
+	void ChangeProfile();
+
+	void RefreshProfileCache();
+
+	void RefreshProfiles(bool refreshCache = false);
+
+	void ActivateProfile(const OBSProfile &profile, bool reset = false);
+	std::vector<std::string>
+	GetRestartRequirements(const ConfigFile &config) const;
+	void ResetProfileData();
+	void CheckForSimpleModeX264Fallback();
+
+public:
+	inline const OBSProfileCache &GetProfileCache() const noexcept
+	{
+		return profiles;
+	};
+
+	const OBSProfile &GetCurrentProfile() const;
+
+	std::optional<OBSProfile>
+	GetProfileByName(const std::string &profileName) const;
+	std::optional<OBSProfile>
+	GetProfileByDirectoryName(const std::string &directoryName) const;
+
+private slots:
+	void on_actionNewProfile_triggered();
+	void on_actionDupProfile_triggered();
+	void on_actionRenameProfile_triggered();
+	void on_actionRemoveProfile_triggered(bool skipConfirmation = false);
+	void on_actionImportProfile_triggered();
+	void on_actionExportProfile_triggered();
+
+public slots:
+	bool NewProfile(const QString &name);
+	bool DuplicateProfile(const QString &name);
+	void DeleteProfile(const QString &profileName);
+
+	// MARK: - OBS Scene Collection Management
+private:
+	OBSSceneCollectionCache collections{};
+
+	void SetupNewSceneCollection(const std::string &collectionName);
+	void SetupDuplicateSceneCollection(const std::string &collectionName);
+	void SetupRenameSceneCollection(const std::string &collectionName);
+
+	const OBSSceneCollection &
+	CreateSceneCollection(const std::string &collectionName);
+	void RemoveSceneCollection(OBSSceneCollection collection);
+
+	bool DuplicateSceneCollection(const QString &name);
+	void DeleteSceneCollection(const QString &name);
+	void ChangeSceneCollection();
+
+	void RefreshSceneCollectionCache();
+
+	void RefreshSceneCollections(bool refreshCache = false);
+	void ActivateSceneCollection(const OBSSceneCollection &collection);
+
+public:
+	inline const OBSSceneCollectionCache &
+	GetSceneCollectionCache() const noexcept
+	{
+		return collections;
+	};
+
+	const OBSSceneCollection &GetCurrentSceneCollection() const;
+
+	std::optional<OBSSceneCollection>
+	GetSceneCollectionByName(const std::string &collectionName) const;
+	std::optional<OBSSceneCollection>
+	GetSceneCollectionByFileName(const std::string &fileName) const;
+
+private slots:
+	void on_actionNewSceneCollection_triggered();
+	void on_actionDupSceneCollection_triggered();
+	void on_actionRenameSceneCollection_triggered();
+	void
+	on_actionRemoveSceneCollection_triggered(bool skipConfirmation = false);
+	void on_actionImportSceneCollection_triggered();
+	void on_actionExportSceneCollection_triggered();
+
+public slots:
+	bool NewSceneCollection(const QString &name);
```

Comment:
It's a little odd to me to start a slot/function name with `New`. I'd expect a verb, such as `Create` or `Add` or even `CreateNew` (redundant though it may be). That said, I see there are private functions that use `Create`, so I'm not sure what to do to both use a verb and avoid confusion.

Reply:
I think this was missed in the latest round of revisions.